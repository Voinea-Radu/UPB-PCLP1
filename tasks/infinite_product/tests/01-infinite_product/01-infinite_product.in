6
1 1
1 2
12 3
12 5
24 5
24 5
