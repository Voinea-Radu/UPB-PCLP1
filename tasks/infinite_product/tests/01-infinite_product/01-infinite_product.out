283
10 3
31.6543836 9.4339811
